package-lock.json

```diff
@@ -0,0 +1,4632 @@
+{
```

Comment:
oclif uses yarn: can you delete this file and install the dep with yarn?

Reply:
done